### PR TITLE
scanner implementation

### PR DESCRIPTION
**Pull request by felipeweb**
https://github.com/prest/prest/issues/149 depends prest/adapters#2

### CODE REVIEW

**Review by avelino — APPROVED** [review]
LGTM

File: testdata/schema.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+DB_HOST=${PREST_PG_HOST:-localhost}
+DB_USER=${PREST_PG_USER:-postgres}
+DB_PORT=${PREST_PG_PORT:-5438}
```
Comment (marioidival) [inline testdata/schema.sh]: 5432 ?????

**Review by marioidival — CHANGES_REQUESTED** [review]
Load correctly `prest.toml` please: https://github.com/prest/controllers/blob/master/sql_test.go#L16
```
--- PASS: TestSelectFromTables (0.05s)
	tables_test.go:125: execute select in a table with array
	tables_test.go:125: execute select in a table without custom where clause
	tables_test.go:125: execute select in a table with count all fields *
	tables_test.go:125: execute select in a table with count function
	tables_test.go:125: execute select in a table with custom where clause
	tables_test.go:125: execute select in a table with custom join clause
	tables_test.go:125: execute select in a table with order clause empty
	tables_test.go:125: execute select in a table with custom where clause and pagination
	tables_test.go:125: execute select in a table with select fields
	tables_test.go:125: execute select in a table with select *
	tables_test.go:125: execute select in a table with group by clause
	tables_test.go:125: Execute select in a table with group by and having clause
	tables_test.go:125: execute select in a view without custom where clause
	tables_test.go:125: execute select in a view with count all fields *
	tables_test.go:125: execute select in a view with count function
	tables_test.go:125: execute select in a view with order function
	tables_test.go:125: execute select in a view with custom where clause
	tables_test.go:125: execute select in a view with custom join clause
	tables_test.go:125: execute select in a view with custom where clause and pagination
	tables_test.go:125: execute select in a view with select fields
	tables_test.go:125: execute select in a table with invalid join clause
	tables_test.go:125: execute select in a table with invalid where clause
	tables_test.go:125: execute select in a table with order clause and column invalid
	tables_test.go:125: execute select in a table with invalid pagination clause
	tables_test.go:125: execute select in a table with invalid where clause
	tables_test.go:125: execute select in a table with invalid count clause
	tables_test.go:125: execute select in a table with invalid order clause
	tables_test.go:125: execute select in a table with invalid fields using group by clause
	tables_test.go:125: execute select in a table with invalid fields using group by and having clause
	tables_test.go:125: execute select in a view with an other column
	tables_test.go:125: execute select in a view with where and column invalid
	tables_test.go:125: execute select in a view with custom join clause invalid
	tables_test.go:125: execute select in a view with custom where clause and pagination invalid
	tables_test.go:125: execute select in a view with order by and column invalid
	tables_test.go:125: execute select in a view with count column invalid
=== RUN   TestInsertInTables
```

**Review by crgimenes — APPROVED** [review]
LGTM

**Pull request merged**